src/Textual/blocks/text.jl

```diff
@@ -26,4 +26,4 @@ FastAI.mockblock(Paragraph())
 struct Paragraph <: Block end
 
 FastAI.checkblock(::Paragraph, ::String) = true
-FastAI.mockblock(::Paragraph) = randstring(" ABCEEFGHIJKLMNOPQESRUVWXYZ 1234567890abcdefghijklmnopqrstynwxyz\n\t.,", 40)
+FastAI.mockblock(::Paragraph) = randstring(" ABCEEFGHIJKLMNOPQESRUVWXYZ 1234567890 abcdefghijklmnopqrstynwxyz\n\t.,", 40)
```

Comment:
`Paragraph` can have variable length, right? Let's put this in `mockblock` so that we don't bake in invariants that don't need to hold: 
```suggestion
FastAI.mockblock(::Paragraph) = randstring(" ABCEEFGHIJKLMNOPQESRUVWXYZ 1234567890 abcdefghijklmnopqrstynwxyz\n\t.,", rand(10:40))
```

Reply:
Yeah, that's more reasonable thing to do.